cargo-shuttle/tests/integration/init.rs

```diff
@@ -1,92 +1,28 @@
-use std::{
-    fs::read_to_string,
-    path::{Path, PathBuf},
-};
+use std::fs::read_to_string;
 
-use cargo_shuttle::{Args, Command, CommandOutcome, InitArgs, ProjectArgs, Shuttle};
+use cargo_shuttle::{Args, Shuttle};
+use clap::Parser;
 use indoc::indoc;
 use tempfile::Builder;
 
-/// creates a `cargo-shuttle` init instance with some reasonable defaults set.
-async fn cargo_shuttle_init(path: PathBuf) -> anyhow::Result<CommandOutcome> {
```

Comment:
It was only after I removed this test that I realised I effectively dropped support for initialising without any framework. We probably still want that, right? Should I add an extra option for that both in the interactive and non-interactive version?

Reply:
While it's not strictly necessary, as it's easy enough to init some framework and clean it out, it is a quality of life feature which is welcome when working with an unsupported framework.
Since there is a possibility to implement custom services, there should probably be a way to bootstrap skeletons for them.

Reply:
Hmm, I agree with @AlexCzar about being able to bootstrap custom services. However, it should not form part of this PR as this PR only focuses on making the init interactive.

Since we will add a custom one eventually, it will be good to have the default back (which we'll then swap for the custom one once the custom one is ready). So we'll still want it, and it seems that an option (`--no-framework`?) will be the best for now

Reply:
Yeah, makes perfect sense to me. I've added that option now.